### DISCUSSION TITLE
不会设置了  感觉还是做减法好点

### DISCUSSION THREAD

**Discussion by SuperBlackWednesday** [hooke007/mpv_PlayKit#203] [Q&A 问答] [answered]
我的电脑只能运行mvtools-2x.vpy 这种倍帧    但是我把0903那个版本的复制粘贴到新版本的VS文件夹   然后在mpv.conf设置成以前一样的命令发现不能用了 .
还有以前0903那个版本可以使用叠加的着色器 ，新版本中使用以前一样的命令不能用了 
就是启动播放器直接加载设置好的补帧和着色器不会设置了，跟以前的方法不一样
新版本的交互界面比以前的强多了  还是希望用这个新版本的交互界面和以前版本的设置方法

**Comment by hooke007** [comment, marked as answer]
1. 以前也早就是减法了，Temp 文件夹的关联内容从未进入 release。

2. vpy只是一个脚本，能不能运行取决是是否已满足前置条件。
参考——
- [维基_vpy](https://github.com/hooke007/MPV_lazy/wiki/2_%E8%AE%BE%E7%BD%AE%E6%96%87%E4%BB%B6%E5%A4%B9%EF%BC%88%E4%BE%BF%E6%90%BA%E5%BC%8F%EF%BC%89#vs%E8%84%9A%E6%9C%AC%E8%B7%AF%E5%BE%84-) （脚本所需依赖声明 #**VS脚本路径**）
- [通用指引中的mvtools示例](https://hooke007.github.io/unofficial/mpv_start.html#vapoursynth) （#**6.VapourSynth**）
- https://github.com/hooke007/MPV_lazy/discussions/66 （#**尝试再追加第三方滤镜**）

3. > 还有以前0903那个版本可以使用叠加的着色器 ，新版本中使用以前一样的命令不能用了

提供的有效信息太少无法判断此问题

**Reply by SuperBlackWednesday** [reply]
我看了 下载了python便携包  还有 VapourSynth 便携包 还有mvtool 覆盖一下  然后任何电影都播放不了了  非专业 不折腾了 我还是老老实实用0903版本吧 等你出个完全版的   .......

**Reply by hooke007** [reply]
> 下载了python便携包 还有 VapourSynth 便携包

对于现有的mpv-lazy只需要脚本和插件，不存在覆盖的操作。

按照当前仓库的目录结构出完整版既无法正常运行，工作量大，包体积也巨大，没有实用意义，因此不会存在这样的版本。

**Reply by SuperBlackWednesday** [reply]
不是那个意思   以前的版本可以用很多的补帧方案例如mvtools-2x.vpy  以及 可以在mpv.conf里面使用glsl-shaders         = "~~/shaders/Anime4K_Clamp_Highlights.glsl;~~/shaders/Anime4K_Restore_CNN_M.glsl;~~/shaders/Anime4K_Upscale_CNN_x2_S.glsl"这种组合命令    
这个新版本的无法启动这些  我还不会安装  
可不可以出一个基于0903版本的升级版呢？ 我想改一个新版本一样的外观 没弄好

**Reply by hooke007** [reply]
> 例如mvtools-2x.vpy

如前所述，只需要添加脚本 mvtools-2x.vpy 和对应的插件 （一共就两文件）

> 这种组合命令

完全没问题，前提你要的着色器确实存在对应路径，或者你没写错

> 基于0903版本的升级版

obs版本终止维护

**Reply by SuperBlackWednesday** [reply]
那个mvtool可以用了  十分感谢  
组合的着色器   还是不行  我的显卡功耗还是10几瓦  大神  你的组合命令是怎么写的 能发过来吗？   
 还有 我每次退出后播放列表不保存  这个是怎么设置的？

**Reply by hooke007** [reply]
```
glsl-shaders = "~~/shaders/Anime4K_Clamp_Highlights.glsl;~~/shaders/Anime4K_Restore_CNN_M.glsl;~~/shaders/Anime4K_Upscale_CNN_x2_S.glsl"
```

另外学会看控制台或者导出日志，这是排错的第一步。

> 每次退出后播放列表不保存

当前的mpv-lazy就没有这个功能
